spec/EditorInserter.spec.js

```diff
@@ -1,17 +1,77 @@
 'use babel';
 
-import EditorInserter from '../lib/editor/EditorInserter';
+import EditorInserter, {
+    EmptyInsertionTextError,
+    MissingEditorError,
+} from '../lib/editor/EditorInserter';
+import sinon from 'sinon';
+import chai from 'chai';
+import { expect } from 'chai';
+
+chai.use(require('sinon-chai'));
+
+class Editor {
+    getSelections() {}
+}
+
+class Selection {
+    getText() {}
+    selectToBeginningOfLine() {}
+    selectRight() {}
+    insertText() {}
+}
```

Comment:
Because `atom` does not seem to be on npm, I must create these scaffold classes to perform stubbing.